{
	"id": "CVE-2021-27102",
	"vulnerabilities": [
		{
			"content": {
				"id": "CVE-2021-27102",
				"description": "Accellion FTA 9_12_411 and earlier is affected by OS command execution via a local web service call. The fixed version is FTA_9_12_416 and later.",
				"severity": [
					{
						"type": "cvss_v2",
						"source": "nvd@nist.gov",
						"cvss_v2": {
							"vector": "AV:L/AC:L/Au:N/C:C/I:C/A:C",
							"base_score": 7.2,
							"nvd_base_severity": "HIGH",
							"temporal_score": 7.2,
							"nvd_temporal_severity": "HIGH",
							"environmental_score": 7.1,
							"nvd_environmental_severity": "HIGH"
						}
					},
					{
						"type": "cvss_v31",
						"source": "nvd@nist.gov",
						"cvss_v31": {
							"vector": "CVSS:3.1/AV:L/AC:L/PR:L/UI:N/S:U/C:H/I:H/A:H",
							"base_score": 7.8,
							"base_severity": "HIGH",
							"temporal_score": 7.8,
							"temporal_severity": "HIGH",
							"environmental_score": 7.8,
							"environmental_severity": "HIGH"
						}
					}
				],
				"cwe": [
					{
						"source": "nvd@nist.gov",
						"cwe": [
							"CWE-78"
						]
					}
				],
				"references": [
					{
						"source": "cve@mitre.org",
						"url": "https://github.com/accellion/CVEs/blob/main/CVE-2021-27102.txt"
					},
					{
						"source": "cve@mitre.org",
						"url": "https://www.accellion.com/products/fta/"
					},
					{
						"source": "nvd.nist.gov",
						"url": "https://nvd.nist.gov/vuln/detail/CVE-2021-27102"
					}
				],
				"published": "2021-02-16T21:15:13.14Z",
				"modified": "2021-02-19T21:26:47.32Z"
			},
			"ecosystems": [
				"cpe"
			]
		}
	],
	"detection": [
		{
			"ecosystem": "cpe",
			"criteria": {
				"operator": "OR",
				"criterias": [
					{
						"operator": "OR",
						"criterias": [
							{
								"operator": "OR",
								"criterias": [
									{
										"operator": "OR",
										"criterions": [
											{
												"vulnerable": true,
												"fix_status": {
													"class": "unknown"
												},
												"package": {
													"cpe": "cpe:2.3:a:accellion:fta:*:*:*:*:*:*:*:*"
												},
												"affected": {
													"type": "unknown",
													"range": [
														{
															"le": "9_12_411"
														}
													]
												}
											},
											{
												"vulnerable": true,
												"fix_status": {
													"class": "unknown"
												},
												"package": {
													"cpe": "cpe:2.3:a:accellion:fta:-:*:*:*:*:*:*:*"
												}
											},
											{
												"vulnerable": true,
												"fix_status": {
													"class": "unknown"
												},
												"package": {
													"cpe": "cpe:2.3:a:accellion:fta:9_12_220:*:*:*:*:*:*:*"
												}
											},
											{
												"vulnerable": true,
												"fix_status": {
													"class": "unknown"
												},
												"package": {
													"cpe": "cpe:2.3:a:accellion:fta:9_12_370:*:*:*:*:*:*:*"
												}
											},
											{
												"vulnerable": true,
												"fix_status": {
													"class": "unknown"
												},
												"package": {
													"cpe": "cpe:2.3:a:accellion:fta:9_12_380:*:*:*:*:*:*:*"
												}
											},
											{
												"vulnerable": true,
												"fix_status": {
													"class": "unknown"
												},
												"package": {
													"cpe": "cpe:2.3:a:accellion:fta:9_12_411:*:*:*:*:*:*:*"
												}
											}
										]
									}
								]
							}
						]
					}
				]
			}
		}
	],
	"data_source": {
		"id": "nvd-api-cve",
		"raws": [
			"vuls-data-raw-nvd-api-cpematch/45e84c18/A7BD1EA0-8D9D-449C-9D0A-1641EC942F77.json",
			"vuls-data-raw-nvd-api-cve/2021/CVE-2021-27102.json"
		]
	}
}
